project/OSGi.scala

```diff
@@ -44,6 +44,43 @@ object OSGi {
 
   val remote = exports(Seq("akka.remote.*"))
 
+  val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"),
+    imports = Seq(optionalResolution("scala.quasiquotes")))
+
+  val httpCore = exports(Seq("akka.http.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.parboiled2.*"),
+      streamAndHttpImport("akka.shapeless.*")))
+
+  val http = exports(Seq("akka.http.impl.server",
+    "akka.http.scaladsl.server.*", "akka.http.javadsl.server.*",
+    "akka.http.scaladsl.client", "akka.http.scaladsl.coding", "akka.http.scaladsl.common",
+    "akka.http.scaladsl.marshalling", "akka.http.scaladsl.unmarshalling"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*"),
+      streamAndHttpImport("akka.parboiled2.*")))
+
+  val httpTestkit = exports(Seq("akka.http.scaladsl.testkit.*", "akka.http.javadsl.testkit.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpSprayJson = exports(Seq("akka.http.scaladsl.marshallers.sprayjson"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpXml = exports(Seq("akka.http.scaladsl.marshallers.xml"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpJackson = exports(Seq("akka.http.javadsl.marshallers.jackson"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val stream = exports(Seq("akka.stream.*"))
+
+  val streamTestkit = exports(Seq("akka.stream.testkit.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*")))
```

Comment:
sanity checking: because project dependencies the plugin understands and will add the imports magically, right?